File: SRC/ToolBox/Cpcform.vb
Public Class CpcForm : Inherits System.Windows.Forms.Form

    Private Col1 As Boolean
    Private Imgauto1 As Boolean
    Private UPD1 As Boolean
    Private handle1 As String
    Private Evant1 As String
    Private couleurfond1 As String
    Private couleurtexte1 As String

    Public Overloads Property HANDLE() As String
        Get
            Return handle1
        End Get
        Set(value As String)
            handle1 = value

        End Set

    End Property
    Public Property COL As Boolean
        Get
            Return Col1
        End Get
        Set(value As Boolean)
            Col1 = value
        End Set
    End Property

    Public Property IMGAUTO As Boolean
        Get
            Return Imgauto1
        End Get
        Set(value As Boolean)
            Imgauto1 = value
        End Set
    End Property
    Public Property UPD As Boolean
        Get
            Return UPD1
        End Get
        Set(value As Boolean)
            UPD1 = value
        End Set
    End Property
    Public Overloads Property EVENT_PATH() As String

        Get
            Return Evant1
        End Get
        Set(value As String)
            Evant1 = value

        End Set

    End Property
    Public Property CouleurFond As String
        Get
            Return couleurfond1
        End Get
        Set(value As String)
            couleurfond1 = value
        End Set
    End Property
    Public Property CouleurTexte As String
        Get
            Return couleurtexte1
        End Get
        Set(value As String)
            couleurtexte1 = value
        End Set
    End Property
    Sub New()

        Col1 = System.Boolean.TrueString
        Imgauto1 = System.Boolean.FalseString
        UPD1 = System.Boolean.TrueString
        HANDLE = "MonHandle"
        EVENT_PATH = ""
        CouleurFond = "200,255,240"
        CouleurTexte = "250,100,100"
    End Sub

    Private Sub InitializeComponent()
        Me.SuspendLayout()
        '
        'CpcForm
        '
        Me.ClientSize = New System.Drawing.Size(284, 262)
        Me.Name = "CpcForm"
        Me.ResumeLayout(False)

    End Sub


End Class

File: SRC/ToolBox/Cpcform.cs
using Microsoft.VisualBasic.CompilerServices;

namespace ToolBox
{
    public class CpcForm : System.Windows.Forms.Form
    {
        private bool Col1;
        private bool Imgauto1;
        private bool UPD1;
        private string handle1;
        private string Evant1;
        private string couleurfond1;
        private string couleurtexte1;

        public new string HANDLE
        {
            get
            {
                return handle1;
            }

            set
            {
                handle1 = value;
            }
        }

        public bool COL
        {
            get
            {
                return Col1;
            }

            set
            {
                Col1 = value;
            }
        }

        public bool IMGAUTO
        {
            get
            {
                return Imgauto1;
            }

            set
            {
                Imgauto1 = value;
            }
        }

        public bool UPD
        {
            get
            {
                return UPD1;
            }

            set
            {
                UPD1 = value;
            }
        }

        public new string EVENT_PATH
        {
            get
            {
                return Evant1;
            }

            set
            {
                Evant1 = value;
            }
        }

        public string CouleurFond
        {
            get
            {
                return couleurfond1;
            }

            set
            {
                couleurfond1 = value;
            }
        }

        public string CouleurTexte
        {
            get
            {
                return couleurtexte1;
            }

            set
            {
                couleurtexte1 = value;
            }
        }

        public CpcForm()
        {
            Col1 = Conversions.ToBoolean(bool.TrueString);
            Imgauto1 = Conversions.ToBoolean(bool.FalseString);
            UPD1 = Conversions.ToBoolean(bool.TrueString);
            HANDLE = "MonHandle";
            EVENT_PATH = "";
            CouleurFond = "200,255,240";
            CouleurTexte = "250,100,100";
        }

        private void InitializeComponent()
        {
            SuspendLayout();
            // 
            // CpcForm
            // 
            ClientSize = new System.Drawing.Size(284, 262);
            Name = "CpcForm";
            ResumeLayout(false);
        }
    }
}